README.md

```diff
@@ -1,27 +1,26 @@
 # CoreOS Documentation
 
-These are publicly accessible versions of the documentation on [coreos.com](http://coreos.com/docs/).
+This repository contains the Markdown source of the documentation on [coreos.com/docs](https://coreos.com/docs/).
 
-Check out the [help-wanted tag](https://github.com/coreos/docs/issues?q=is%3Aopen+label%3Ahelp-wanted) if you're not sure what to help out with.
+## Contributing
 
-1. Fork this repo
-2. Send a Pull Request
-3. We will get your fix up on [coreos.com](http://coreos.com/docs/)
+The CoreOS documentation is released under the [Apache 2.0 License](LICENSE), and we welcome contributions. Check out the [help-wanted tag](https://github.com/coreos/docs/issues?q=is%3Aopen+label%3Ahelp-wanted) in this project's Issues list for good places to start participating.
 
-Thanks for your contributions and fixes!
+Submit fixes and additions in the form of `git` *Pull Requests* (PRs). The general process is the typical git fork-branch-PR-review-merge cycle:
 
-## Translations
+1. Fork this repository into your git workspace and/or github account
+2. Make changes in a topic branch off your fork's `master`
```

Comment:
This is unnecessarily prescriptive; I submit pull requests from my personal `master` branch all the time